File: filterx-matrix-20260820/projects/fastapi-sqlalchemy__angular/seed.py
from app.database import Base, SessionLocal, engine
from app.models.author import Author
from app.models.book import Book

Base.metadata.drop_all(bind=engine); Base.metadata.create_all(bind=engine)
db = SessionLocal()
ada, bob = Author(name='Ada'), Author(name='Bob')
db.add_all([ada, bob]); db.flush()
db.add_all([Book(title='Alpha Filtering', genre='Tech', price=10, note='first', author=ada), Book(title='Beta Search', genre='Tech', price=30, note=None, author=bob), Book(title='Gamma Grouping', genre='Business', price=40, note='last', author=bob)])
db.commit(); db.close()

File: filterx-matrix-20260820/projects/express-prisma__angular/seed.mjs
import { PrismaClient } from '@prisma/client';
const db = new PrismaClient();
await db.book.deleteMany(); await db.author.deleteMany();
await db.author.create({data:{name:'Ada',books:{create:[{title:'Alpha Filtering',genre:'Tech',price:'10.00',note:'first'}]}}});
await db.author.create({data:{name:'Bob',books:{create:[{title:'Beta Search',genre:'Tech',price:'30.00'},{title:'Gamma Grouping',genre:'Business',price:'40.00',note:'last'}]}}});
await db.$disconnect();
